.classpath

```diff
@@ -6,6 +6,7 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="src" path="main/java"/>
```

Comment:
.classpath should not be changed. We are not interrested in changing this IDE specific file..add it to gitignore